packages/block-editor/src/components/inserter/style.scss

```diff
@@ -1,5 +1,5 @@
 $block-inserter-preview-height: 350px;
-$block-inserter-width: 350px;
+$block-quick-inserter-width: 350px;
```

Comment:
Renaming this as I kept getting confused since this variable was only applicable to the quick inserter (the inline inserter in the canvas)

Reply:
Worth an inline comment as well, defining what that is (for future reference?)